classes/DirtEvent.sc

```diff
@@ -67,6 +83,7 @@ DirtEvent {
 		var accelerate = ~accelerate.value;
 		var avgSpeed, endSpeed;
 		var useUnit;
+		~release = ~release.value ? 0.01;
```

Comment:
default values are usually given in the default event. Unless you need to caclulate something with `~release` here, also the `.value` will be called automatically.